src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -39,18 +49,57 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
+        let content: string = '';
+
+        if (args.options.output && args.options.output.toLowerCase() === 'json') {
+          let reportdata: any = this.getReport(res);
+          content = JSON.stringify(reportdata);
+        }
+        else {
+          content = res;
+        }
+
+        if (!args.options.outputFilePath) {
+          cmd.log(content);
+        }
+        else {
+          fs.writeFileSync(args.options.outputFilePath, content, 'utf8');
+          if (this.verbose) {
+            cmd.log(`File saved to path '${args.options.outputFilePath}'`);
+          }
+        }
 
         cb();
       }, (err: any): void => this.handleRejectedODataJsonPromise(err, cmd, cb));
-}
+  }
+
+  private getReport(res: string): any {
+    const rows = res.split('\n');
```

Comment:
Please add types to this and following lines.

Reply:
Done. This is fixed